frontend/language/src/en.json

```diff
@@ -25,7 +25,9 @@
   "header_form.factory.form.builder.link": "Form Builder",
   "header_form.factory.home.link": "Home",
   "header_form.factory.title.link": "Form Factory",
+  "login_page.auth.error": "You must be logged in to access the requested page.",
```

Comment:
I think we can simplify the message to "Please log in, or create an account if you're new here."